toscssModule.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
```

Comment:
Are you sure you want to commit this?

Reply:
not sure may be just put the script on a gist would be enough